pkgs/servers/sql/postgresql/packages.nix

```diff
@@ -44,4 +44,6 @@ self: super: {
     pgrouting = super.callPackage ./ext/pgrouting.nix { };
 
     pg_partman = super.callPackage ./ext/pg_partman.nix { };
+
+    pg-safeupdate = super.callPackage ./ext/pg-safeupdate.nix { };
```

Comment:
Can we rename this as `pg_safeupdate` to match the naming scheme of all the other extensions here?

Reply:
Sure. The extension name has that dash(`pg-safeupdate`) so I left it like that but I don't see a problem with changing it.

Reply:
@thoughtpolice Renaming done!